### PR TITLE
feat: only allow ropsten network

### PR DESCRIPTION
**Pull request by martink-rsa**
Added network to the global net state
Added check for Metamask to be on the ropsten network

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -41,24 +43,33 @@ function BridgeApp(props: Props) {
   }, []);
 
   // Check if Network has been started
-  if (!isConnected(netClient)) {
+  if (!isConnected(net.client)) {
     return <p style={{ textAlign: 'center' }}>Connecting Network</p>;
   }
 
+  // Ensure that network is ropsten
+  if (net.network !== 'ropsten') {
```
Comment (musnit) [inline src/App.tsx:51]: can we put this ('ropsten') in a config file that can be changed in different environments (eg: in future we may want to deploy a version that supports mainnet)

**Review by musnit — APPROVED** [review]

**Review by jakai7 — APPROVED** [review]
Awesome changes.

File: src/App.tsx (hunk 1 above)
Comment (martink-rsa) [reply src/App.tsx]: It uses an environment value now with a fallback to `ropsten`

File: .env-template [hunk 2]
```diff
@@ -0,0 +1 @@
+GATSBY_PERMITTED_METAMASK_NETWORK=
```
Comment (musnit) [inline .env-template:1]: why gatsby :)?

Comment (martink-rsa) [reply .env-template]: GATSBY was removed and this is resolved.

**Pull request merged**